LDAR_Sim/src/ldar_sim_main.py

```diff
@@ -54,6 +55,11 @@
     root_dir = Path(os.path.dirname(os.path.realpath(__file__))).parent
     os.chdir(root_dir)
 
+    src_dir = root_dir / 'src'
```

Comment:
Can this not be done with pathlib?

Reply:
Could not figure out how to use pathlib in another way.